music/demos/transcription.ts

```diff
@@ -47,6 +74,11 @@ async function transcribe() {
   writeTimer('transcription-time', start);
   writeNoteSeqs('transcription-results', [ns], undefined, true);
   oaf.dispose();
+
+  document.getElementById('ns-match').innerHTML =
+      notesMatch(ns.notes, expectedNs.notes) ?
+      '<font color="green">TRUE</font>' :
```

Comment:
Oh no, we don't use `<font>` anymore. How about:
`<span style="color:green>TRUE</span>` (and the same for false)